hm-post-repeat.php

```diff
@@ -144,7 +152,7 @@ function save_post_repeating_status() {
 	}
 
 	else {
-		update_post_meta( get_the_id(), 'hm-post-repeat', true );
+		update_post_meta( get_the_id(), 'hm-post-repeat', $_POST['hm-post-repeat'] );
```

Comment:
Yep, will add a check to make sure that a valid schedule has been chosen, also to prevent a manual change to the HTML (which would basically allow *any* input if only the string itself were sanitised).